mock-data/insertData.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../database/index');
+const mock = require('./generateData');
+
+const userAccounts = mock.generateFirstName(755);
+const ownerAccounts = mock.generateFirstName(100);
+const listingInfo = mock.generateAllListingInfo(100);
+const allBookings = mock.generateAllBookings(100, 755);
+const listingReviews = mock.generateAllReviews(100, 755);
+
+userAccounts.forEach((user) => {
```

Comment:
It's ok here because mysql takes care of the writing stream in the background in order. (For future purpose, better use asynchronous function pattern (async/await, promises, callback etc.) to suggest db.insertMockData are asynchronous functions, and they are to execute one after another.)